src/main/java/com/synopsys/integration/rest/client/IntHttpClient.java

```diff
@@ -329,6 +330,11 @@ private Response handleClientExecution(HttpUriRequest request, HttpContext httpC
             Response response = new DefaultResponse(request, client, closeableHttpResponse);
             logResponseHeaders(closeableHttpResponse);
             if (response.isStatusCodeError()) {
+                // what's returned by getContentString appears to be a JSON 
+                // structure in string form.  We have to make this call here in
+                // order to have it available in the response object when it 
+                // handles the error message.
+                String responseContent = response.getContentString();
```

Comment:
possible to just invoke the method and not do the assignment? I don't think it is used.